tools/generate_from_protos.sh

```diff
@@ -10,6 +10,11 @@ third_party_dir="${script_dir}/../build/default/third_party"
 protoc_binary="${third_party_dir}/install/bin/protoc"
 protoc_grpc_binary="${third_party_dir}/install/bin/grpc_cpp_plugin"
 
+function snake_case_to_camel_case {
```

Comment:
I thought I had changed `followme` to `follow_me`, but then I would have had to add something like this :thinking:. Weird :laughing:

Reply:
I'm confused because `follow_me` is not used in this script yet.

Reply:
Yes, maybe I did that in the `proto` submodule actually, for Java. So I'm confused :laughing: